src/main/kotlin/snyk/common/SnykCachedResults.kt

```diff
@@ -203,6 +198,33 @@ class SnykCachedResults(
                             project,
                         )
                 }
+
+                override fun onPublishDiagnostics(
+                    product: String,
+                    snykFile: SnykFile,
+                    issueList: List<ScanIssue>
+                ) {
+
+                    when (product) {
+                        LsProductConstants.OpenSource.value -> {
+                            currentOSSResultsLS[snykFile] = emptyList()
```

Comment:
I think this line is redundant to the following line, if no issues are in the transmitted list.